MCprep_addon/world_tools.py

```diff
@@ -310,8 +312,13 @@ def execute(self, context):
 		obj_import_mem_msg = (
 			"Memory error during OBJ import, try exporting a smaller world")
 		try:
-			res = bpy.ops.import_scene.obj(
-				filepath=self.filepath, use_split_groups=True)
+			res = None
+			if util.min_bv((3, 5)):
+				res = bpy.ops.wm.obj_import(
+					filepath=self.filepath, use_split_groups=True) 
```

Comment:
I think this diff hunk can be reverted. This "use_split_groups" works in 3.5 and equally in 3.4 and earlier, as written this would make 3.4 and earlier stop splitting out by objects from the source, which would not be good.

Reply:
Not sure what you mean, though I did catch the use of the wrong operator for 3.4 and below. In my testing, `use_split_groups` doesn't seem to work in 3.4